plugins/modules/ec2_snapshot.py

```diff
@@ -364,6 +363,7 @@ def create_snapshot_ansible_module():
         required_if=required_if,
         required_one_of=required_one_of,
         required_together=required_together,
+        supports_check_mode=True
```

Comment:
```suggestion
        supports_check_mode=True,
```

If you add the extra `,` on the end of the line here, then when someone adds another entry they don't need to change an unrelated line.